Applications/CMC/test/testCMCArm26_Thelen.cpp

```diff
@@ -41,10 +41,27 @@ void testCMCArm26() {
     Storage *standard = new Storage();
     cmc.getModel().formStateStorage(temp, *standard);
 
-    Array<double> rms_tols(0.02, 2*2+2*6); // activations within 2%, angles within .6 degrees
+    Array<double> rms_tols(0, 2*2+2*6); 
     const string& muscleType = cmc.getModel().getMuscles()[0].getConcreteClassName();
```

Comment:
The original tolerances are justifiable. Id they could be tightened to 1% or 0.5 degree that would be more than sufficient. It is not necessary to ratchet down to detect minor changes likely do to different math libraries in different environments.